feat(curriculum): Add fCC Authors Page Project to JS Cert (#49655)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/6419aa46a11acb514e1731e6.md

---
id: 6419aa46a11acb514e1731e6
title: Step 4
challengeType: 0
dashedName: step-4
---

# --description--

> The data you get from a `GET` request is not usable at first. To make the data usable, you can use the `json()` on the response with the `.then()` method. If you expand the `Prototype` of the `Response` object in the console, you will see the`json()` method there.
> 
> Remove the `console.log(res)` and let the `.then()` return `res.json()` instead – so you can parse the `Response` object to JSON.

Reviewer: ```suggestion
The data you get from a `GET` request is not usable at first. To make the data usable, you can use the `.json()` method on the `Response` object to parse it into JSON. If you expand the `Prototype` of the `Response` object in the console, you will see the `.json()` method there.

Remove `console.log(res)` and return `res.json()` instead.
```
